modules/rgbd/test/test_tsdf.cpp

```diff
@@ -466,7 +466,10 @@ void valid_points_test(bool isHashTSDF)
         waitKey(20000);
     }
 
-    float percentValidity = float(profile) / float(anfas);
+    float percentValidity;
+    if (profile == 0)    percentValidity = -0.5;
+    else if (anfas == 0) percentValidity = 0;
```

Comment:
Why to distinguish `profile` from `anfas` cases? Both zeros mean that test is failed.

Reply:
profile and anfas are the different poses of the scene, 
if no points in anfas   1 type of error
if no points in profile 2 type of error